src/main/java/com/solvd/carina/demo/mobile/saucedemomobile/android/LoginScreen.java

```diff
@@ -0,0 +1,53 @@
+package com.solvd.carina.demo.mobile.saucedemomobile.android;
+
+import com.solvd.carina.demo.mobile.saucedemomobile.common.LoginScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.ProductsScreenBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = LoginScreenBase.class)
+public class LoginScreen extends LoginScreenBase implements IMobileUtils {
```

Comment:
LoginScreenBase extends AbstractSauceDemoScreenBase which implements IMobileUtils. So, IMobileUtils can be removed here?